### ISSUE TITLE
Cannot build for iOS simulator using Swift Package Manager

### ISSUE THREAD

**Issue by sterien7** [pvzig/SlackKit#194]
I am using Xcode 12.4 Swift 5 and I am trying to run my app to an iOS simulator.
I have followed the installation guide but I am getting the following error

```
ld: building for iOS Simulator, but linking in dylib built for macOS, file '/usr/local/Cellar/libressl/3.2.3/lib/libssl.dylib' for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

Any ideas?

**Comment by pvzig** [comment]
This should be resolved by release [4.6.1](https://github.com/pvzig/SlackKit/releases/tag/4.6.1).

**Comment by seadiem** [comment]
How did you fix it?
